x-pack/plugins/observability_solution/infra/common/snapshot_metric_i18n.ts

```diff
@@ -12,8 +12,12 @@ import { SnapshotMetricType } from '@kbn/metrics-data-access-plugin/common';
 // Lowercase versions of all metrics, for when they need to be used in the middle of a sentence;
 // these may need to be translated differently depending on language, e.g. still capitalizing "CPU"
 const TranslationsLowercase = {
+  CPUUsageTotal: i18n.translate('xpack.infra.waffle.metricOptions.cpuUsageTotalText', {
+    defaultMessage: 'CPU usage (total)',
+  }),
+
   CPUUsage: i18n.translate('xpack.infra.waffle.metricOptions.cpuUsageText', {
-    defaultMessage: 'CPU usage',
+    defaultMessage: 'CPU usage (user/system)',
```

Comment:
Hey @MiriamAparicio , @roshan-elastic  and I had a chat we think it will be easier for users to understand if the labels are:

old: `CPU Usage (Legacy)`
new: `CPU Usage`

Sorry for the last minute change.

Reply:
Oh, just seen this, I just added all the screenshots 🤦‍♀️